Faraday_Proxy_Tools/FaradayIO/commandmodule.py

```diff
@@ -291,7 +290,7 @@ def packet_gpio_gps_standby_enable():
 
     .. todo:: This function needs updating to the new create_gpio_command_packet()
     """
-    return create_gpio_command_packet(3, gpio_allocation.GPS_STANDBY, 0)
```

Comment:
What's up with gpio_allocations moving to gpioallocations? This seems fine it just doesn't appear to me that these two are renamed anywhere prior to this? @reillyeon @kb1lqd